src/main/java/step3/domain/RacingCarGame.java

```diff
@@ -0,0 +1,54 @@
+package step3.domain;
+
+import step3.strategy.MoveStrategy;
+import step3.strategy.RandomMove;
+import step3.ui.InputView;
+import step3.ui.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public final class RacingCarGame {
+    private List<RacingCar> racingCars;
```

Comment:
`컬렉션 프레임워크`를 이용해서 구현을 해주셨군요 💯 💯 💯   
사실 이대로 진행을 해도 비즈니스 로직을 구현하는데 무리는 없지만  
더 좋은 방법이 있어서 코멘트를 남겼습니다 :) 

```suggestion
    private RacingCars racingCars;
```   
제가 `TDD`를 진행하면서 배운 것 중 가장 좋은 기법 중 하나라고 생각하는 `일급 컬렉션`이라는 기법이 있어요     
아래 참고 사이트와 제가 정리한 내용이 있으니 확인 부탁드릴게요 😉

[참고 사이트1](https://jojoldu.tistory.com/412)     
[참고 사이트2](https://github.com/my-next-step/lecture-and-retrospect/blob/main/%EC%9D%BC%EA%B8%89%20%EC%BB%AC%EB%A0%89%EC%85%98.md)